financialaid/constants.py

```diff
@@ -0,0 +1,224 @@
+"""
+Constants for financialaid
+"""
+
+
+DEFAULT_INCOME_THRESHOLD = 100000
+COUNTRY_INCOME_THRESHOLDS = {
```

Comment:
For countries which exist in both lists it looks like the thresholds all match up as expected